config.yml

```diff
@@ -0,0 +1,6 @@
+slug: gql
```

Comment:
@jmertic How is this slug used? In general we try and avoid the abbreviation "gql" for GraphQL (the [brand guidelines](https://graphql.org/brand/) say not to use it) because, among other things, it clashes with [Graph Query Language](https://en.wikipedia.org/wiki/Graph_Query_Language).

Reply:
Hey - this is an internal LFX slug designation. It might make sense since this shows publicly in a few URLs ( such as https://zoom-lfx.platform.linuxfoundation.org/meetings/gql?view=month ) that we should change it. I'll chat with @jorydotcom on this.

Reply:
@benjie Could we merge in as-is for now, and earmark to fix later? Changing slugs is a process and we are trying to update the build system for the landscape which is currently broken.

Reply:
Thanks! Not a huge deal if it's pointing to existing stuff, I just do my best to ensure it's not being used in any _new_ locations :)

Reply:
Sounds good! I will see internally if we could change it, but it takes a while as it interfaces to multiple services.